README.rst

```diff
@@ -52,3 +52,23 @@ Comparing ``rdopkg patch`` with this ``tar-changes`` script:
 
 .. _rdopkg: https://github.com/softwarefactory-project/rdopkg
 .. _Limited: https://github.com/softwarefactory-project/rdopkg/issues/169
+
+Understanding branches
+========================
+
+We build RH Ceph Storage in ``ceph-*`` Git branches.
+**dist-git branches** follow the form ``<product-name>-<product-version>-<OS>-<OS version>``.
+
+For RH Ceph Storage 5, we support both RHEL 8 and RHEL 9, so we have the following branches:
```

Comment:
When I view the rendered `README.rst` on GitHub, I see that it does not format this list properly. We need a newline here.
```suggestion
For RH Ceph Storage 5, we support both RHEL 8 and RHEL 9, so we have the following branches:

```